adam-core/src/main/scala/org/bdgenomics/adam/converters/TranscriptEffectConverter.scala

```diff
@@ -132,26 +170,98 @@ object VariantAnnotations extends Serializable with Logging {
     Seq(te.build())
```

Comment:
Unrelated to this PR, as this line is unchanged, but whenever possible, I prefer `Iterable` to `Seq` unless you need random lookup by index.